frontend/app/src/components/status/NotificationIndicator.vue

```diff
@@ -13,16 +13,16 @@
           <template #badge>
             <span>{{ count }}</span>
           </template>
-          <v-btn
-            icon
+          <menu-tooltip-button
+            tooltip="Notifications"
+            class-name="secondary--text text--lighten-2"
+            :on-menu="on"
             retain-focus-on-click
```

Comment:
Are we sure about this specific behavior? I noticed that it's used only here

Reply:
The "retain focus on click"? I think it's because of the badges component. If you clicked this and it didn't use retain-focus the menu styling is a little different than the other menus. Can you double check?

Reply:
I don't see a difference from the style. According to the documentation, this is about keep the focus on the button when the men is opened 🤔

Reply:
Maybe @kelsos has more context about that point here?

Reply:
No unfortunately, I honestly don't remember, maybe it is just some copy paste from the documentation. If it doesn't make a difference from a functionality or style point then we can remove it.

Reply:
Yes, it's what I said (just confirmed):
Here is the notification button (in current `develop`) with `retain-focus-on-click` after you click:
![image](https://user-images.githubusercontent.com/27592/90170418-4f1c9300-dda0-11ea-9797-01e0670d4115.png)

Here it is _without_ `retain-focus-on-click` (the background of the button is not "focused")
![image](https://user-images.githubusercontent.com/27592/90170391-47f58500-dda0-11ea-9e58-d295dcd3712c.png)

Reply:
Oh ok, I didn't noticed that, indeed, so we have to keep it 👍